webpack.common.config.js

```diff
@@ -11,7 +11,16 @@ var LOCAL_PACKAGES = resolve(__dirname, 'packages/');
 var COURAGE_DIST = `${LOCAL_PACKAGES}/@okta/courage-dist/esm`;
 
 // Return a function so that all consumers get a new copy of the config
-module.exports = function(outputFilename, mode = 'development') {
+module.exports = function({
+  entry,
+  outputFilename,
+  mode = 'development',
+  engine = '',
+  cdn = true
+}) {
+
+  // normalize entry so it is always an array
+  entry = Array.isArray(entry) ? entry : [entry];
```

Comment:
```suggestion
  entry = typeof entry === 'string' ? [entry] : entry;
```

Reply:
What is the problem if the entry is an object and not a string?